docs/ops/movement/Unique_10.md

```diff
@@ -0,0 +1,111 @@
+# Unique  {#openvino_docs_ops_movement_Unique_10}
+
+**Versioned name**: *Unique-10*
+
+**Category**: *Data movement*
+
+**Short description**: *Unique* finds unique elements of a given input tensor. Depending on the operator's attributes it either looks for unique values globally in the whole (flattened) tensor or performs the search along the specified axis.
+
+**Detailed description**
+The operator can either work in elementwise mode searching for unique values in the whole tensor or it can consider slices of the input tensor along the specified axis. This way the op is able to find unique subtensors in the input data. Except for the unique elements the operator also produces the indices of the unique elements in the input tensor and the number of occurences of unique elements in the input data.
+
+**Attributes**:
+
+* *sorted*
+
+  * **Description**: controls whether the unique elements in the output tensor are sorted in ascending order.
+  * **Range of values**:
+    * false - output tensor's elements are not sorted
+    * true - output tensor's element are sorted
+  * **Type**: boolean
+  * **Default value**: true
+  * **Required**: *no*
+
+* *index_element_type*
+
+  * **Description**: controls the data type of the output tensors containing indices.
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i64"
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: A tensor of type *T* and arbitrary shape. **Required.**
+* **2**: A tensor of type *T_AXIS*. The allowed tensor shape is 1D with a single element or a scalar. **Optional**
+  When provided this input is used to "divide" the input tensor into slices along the specified axis before unique elements processing starts. When this input is not provided the operator works on a flattened version of the input tensor (elementwise processing).
+
+**Outputs**
+
+* **1**: The output tensor containing unique elements (individual values or subtensors). This tensor's type matches the type of the first input tensor: *T*. The values in this tensor are either sorted ascendingly or maintain the same order as in the input tensor. The shape of this output depends on the values of the input tensor and will very often be dynamic. Please refer to the article describing how [Dynamic Shapes](https://docs.openvino.ai/latest/openvino_docs_OV_UG_DynamicShapes.html) are handled in OpenVINO.
+* **2**: The output tensor containing indices of the locations of unique elements. The indices map the elements in the first output tensor to their locations in the input tensor. The index always points to the first occurence of a given unique output element in the input tensor. This is a 1D tensor with type *T_IND*.
```

Comment:
I think we should tell `This is a 1D tensor with type *index_element_type*.` and then *T_IND* is not needed. The next output is the same but I am not sure if we need a separate type attribute for it.